keyboards/1upkeyboards/1upsuper16v3/info.json

```diff
@@ -0,0 +1,196 @@
+
+        { "flags": 4, "matrix": [0, 8], "x": 28, "y": 40 },
+        { "flags": 4, "matrix": [0, 9], "x": 84, "y": 40 },
+
+        { "flags": 2, "x": 4, "y": 52 },
+        { "flags": 2, "x": 4, "y": 60 },
+        { "flags": 2, "x": 14, "y": 63 },
+
+        { "flags": 4, "matrix": [0, 12], "x": 28, "y": 56 },
+
+        { "flags": 2, "x": 42, "y": 63 },
+        { "flags": 2, "x": 70, "y": 63 },
+
+        { "flags": 4, "matrix": [0, 13], "x": 84, "y": 56 },
+
+        { "flags": 2, "x": 99, "y": 63 },
+        { "flags": 2, "x": 126, "y": 63 },
+
+        { "flags": 4, "matrix": [0, 14], "x": 140, "y": 56 },
+
+        { "flags": 2, "x": 154, "y": 63 },
+        { "flags": 2, "x": 182, "y": 63 },
+
+        { "flags": 4, "matrix": [0, 15], "x": 196, "y": 56 },
+
+        { "flags": 2, "x": 210, "y": 63 },
+        { "flags": 2, "x": 220, "y": 60 },
+        { "flags": 2, "x": 220, "y": 52 },
+        { "flags": 2, "x": 220, "y": 44 },
+
+        { "flags": 4, "matrix": [0, 11], "x": 196, "y": 40 },
+        { "flags": 4, "matrix": [0, 10], "x": 140, "y": 40 },
+
+        { "flags": 2, "x": 220, "y": 36 },
+        { "flags": 2, "x": 220, "y": 28 },
+
+        { "flags": 4, "matrix": [0, 7], "x": 196, "y": 24 },
+        { "flags": 4, "matrix": [0, 6], "x": 140, "y": 24 },
+
+        { "flags": 2, "x": 220, "y": 20 },
+        { "flags": 2, "x": 220, "y": 12 },
+
+        { "flags": 4, "matrix": [0, 3], "x": 196, "y": 8 },
+        { "flags": 4, "matrix": [0, 2], "x": 140, "y": 8 },
+
+        { "flags": 2, "x": 220, "y": 4 },
+        { "flags": 2, "x": 210, "y": 1 },
+        { "flags": 2, "x": 182, "y": 1 },
+        { "flags": 2, "x": 154, "y": 1 },
+        { "flags": 2, "x": 126, "y": 1 }
+
+        ]
+    },
+    "community_layouts": [ "ortho_4x4" ],
+    "layouts": {
+        "LAYOUT_ortho_4x4": {
+            "layout": [
+                { "label": "00", "matrix": [0, 0], "x": 0, "y": 0 },
+                { "label": "01", "matrix": [0, 1], "x": 1, "y": 0 },
+                { "label": "02", "matrix": [0, 2], "x": 2, "y": 0 },
+                { "label": "03", "matrix": [0, 3], "x": 3, "y": 0 },
+                { "label": "10", "matrix": [0, 4], "x": 0, "y": 1 },
+                { "label": "11", "matrix": [0, 5], "x": 1, "y": 1 },
+                { "label": "12", "matrix": [0, 6], "x": 2, "y": 1 },
+                { "label": "13", "matrix": [0, 7], "x": 3, "y": 1 },
+                { "label": "20", "matrix": [0, 8], "x": 0, "y": 2 },
+                { "label": "21", "matrix": [0, 9], "x": 1, "y": 2 },
+                { "label": "22", "matrix": [0, 10], "x": 2, "y": 2 },
+                { "label": "23", "matrix": [0, 11], "x": 3, "y": 2 },
+                { "label": "30", "matrix": [0, 12], "x": 0, "y": 3 },
+                { "label": "31", "matrix": [0, 13], "x": 1, "y": 3 },
+                { "label": "32", "matrix": [0, 14], "x": 2, "y": 3 },
+                { "label": "33", "matrix": [0, 15], "x": 3, "y": 3 }
```

Comment:
entirely optional - puts the direct pin matrix into same shape as physical layout
```suggestion
            ["GP23", "GP24", "GP4", "GP5"],
            ["GP19", "GP27", "GP1", "GP8"],
            ["GP18", "GP28", "GP0", "GP9"],
            ["GP17", "GP16", "GP15", "GP11" ]
        ]
    },
    "ws2812": {
        "pin": "GP29",
        "driver": "vendor"
    },
    "rgb_matrix": {
        "animations": {
            "gradient_up_down": true,
            "gradient_left_right": true,
            "breathing": true,
            "band_sat": true,
            "band_val": true,
            "band_pinwheel_sat": true,
            "band_pinwheel_val": true,
            "band_spiral_sat": true,
            "band_spiral_val": true,
            "cycle_all": true,
            "cycle_left_right": true,
            "cycle_up_down": true,
            "rainbow_moving_chevron":true,
            "cycle_out_in": true,
            "cycle_out_in_dual": true,
            "cycle_pinwheel": true,
            "cycle_spiral": true,
            "dual_beacon": true,
            "rainbow_beacon": true,
            "rainbow_pinwheels": true,
            "raindrops": true,
            "jellybean_raindrops": true,
	        "hue_breathing": true,
            "hue_pendulum": true,
            "hue_wave": true,
            "pixel_rain": true,
            "pixel_flow": true,
            "typing_heatmap": true,
            "digital_rain": true,
            "solid_reactive_simple": true,
	        "solid_reactive": true,
            "solid_reactive_wide": true,
            "solid_reactive_multiwide": true,
            "solid_reactive_cross": true,
            "solid_reactive_multicross": true,
            "solid_reactive_nexus": true,
            "solid_reactive_multinexus": true,
            "splash": true,
            "multisplash": true,
            "solid_splash": true,
            "solid_multisplash": true
        },
        "driver": "WS2812",
        "layout": [

        { "flags": 2, "x": 99, "y": 1 },
        { "flags": 2, "x": 70, "y": 1 },
        { "flags": 2, "x": 42, "y": 1 },
        { "flags": 2, "x": 14, "y": 1 },
        { "flags": 2, "x": 4, "y": 4 },
        { "flags": 2, "x": 4, "y": 12 },

        { "flags": 4, "matrix": [0, 0], "x": 28, "y": 8 },
        { "flags": 4, "matrix": [0, 1], "x": 84, "y": 8 },

        { "flags": 2, "x": 4, "y": 20 },
        { "flags": 2, "x": 4, "y": 28 },

        { "flags": 4, "matrix": [0, 4], "x": 28, "y": 24 },
        { "flags": 4, "matrix": [0, 5], "x": 84, "y": 24 },

        { "flags": 2, "x": 4, "y": 36 },
        { "flags": 2, "x": 4, "y": 44 },

        { "flags": 4, "matrix": [0, 8], "x": 28, "y": 40 },
        { "flags": 4, "matrix": [0, 9], "x": 84, "y": 40 },

        { "flags": 2, "x": 4, "y": 52 },
        { "flags": 2, "x": 4, "y": 60 },
        { "flags": 2, "x": 14, "y": 63 },

        { "flags": 4, "matrix": [0, 12], "x": 28, "y": 56 },

        { "flags": 2, "x": 42, "y": 63 },
        { "flags": 2, "x": 70, "y": 63 },

        { "flags": 4, "matrix": [0, 13], "x": 84, "y": 56 },

        { "flags": 2, "x": 99, "y": 63 },
        { "flags": 2, "x": 126, "y": 63 },

        { "flags": 4, "matrix": [0, 14], "x": 140, "y": 56 },

        { "flags": 2, "x": 154, "y": 63 },
        { "flags": 2, "x": 182, "y": 63 },

        { "flags": 4, "matrix": [0, 15], "x": 196, "y": 56 },

        { "flags": 2, "x": 210, "y": 63 },
        { "flags": 2, "x": 220, "y": 60 },
        { "flags": 2, "x": 220, "y": 52 },
        { "flags": 2, "x": 220, "y": 44 },

        { "flags": 4, "matrix": [0, 11], "x": 196, "y": 40 },
        { "flags": 4, "matrix": [0, 10], "x": 140, "y": 40 },

        { "flags": 2, "x": 220, "y": 36 },
        { "flags": 2, "x": 220, "y": 28 },

        { "flags": 4, "matrix": [0, 7], "x": 196, "y": 24 },
        { "flags": 4, "matrix": [0, 6], "x": 140, "y": 24 },

        { "flags": 2, "x": 220, "y": 20 },
        { "flags": 2, "x": 220, "y": 12 },

        { "flags": 4, "matrix": [0, 3], "x": 196, "y": 8 },
        { "flags": 4, "matrix": [0, 2], "x": 140, "y": 8 },

        { "flags": 2, "x": 220, "y": 4 },
        { "flags": 2, "x": 210, "y": 1 },
        { "flags": 2, "x": 182, "y": 1 },
        { "flags": 2, "x": 154, "y": 1 },
        { "flags": 2, "x": 126, "y": 1 }

        ]
    },
    "community_layouts": [ "ortho_4x4" ],
    "layouts": {
        "LAYOUT_ortho_4x4": {
            "layout": [
                { "label": "00", "matrix": [0, 0], "x": 0, "y": 0 },
                { "label": "01", "matrix": [0, 1], "x": 1, "y": 0 },
                { "label": "02", "matrix": [0, 2], "x": 2, "y": 0 },
                { "label": "03", "matrix": [0, 3], "x": 3, "y": 0 },
                { "label": "10", "matrix": [1, 0], "x": 0, "y": 1 },
                { "label": "11", "matrix": [1, 1], "x": 1, "y": 1 },
                { "label": "12", "matrix": [1, 2], "x": 2, "y": 1 },
                { "label": "13", "matrix": [1, 3], "x": 3, "y": 1 },
                { "label": "20", "matrix": [2, 0], "x": 0, "y": 2 },
                { "label": "21", "matrix": [2, 1], "x": 1, "y": 2 },
                { "label": "22", "matrix": [2, 2], "x": 2, "y": 2 },
                { "label": "23", "matrix": [2, 3], "x": 3, "y": 2 },
                { "label": "30", "matrix": [3, 0], "x": 0, "y": 3 },
                { "label": "31", "matrix": [3, 1], "x": 1, "y": 3 },
                { "label": "32", "matrix": [3, 2], "x": 2, "y": 3 },
                { "label": "33", "matrix": [3, 3], "x": 3, "y": 3 }
```